crypto/rand/rand_uniform.c

```diff
@@ -85,7 +85,7 @@ uint32_t ossl_rand_uniform_uint32(OSSL_LIB_CTX *ctx, uint32_t upper, int *err)
         if (f < f2)
             return i + 1;
         /* For not all 1 bits, there is no carry so return the result */
-        if (unlikely(f != 0xffffffff))
+        if (likely(f != 0xffffffff))
```

Comment:
were we going to change `likely()` and `unlikely()` to `ossl_*likely()` at some point?

Reply:
(not in this PR, of course)

Reply:
Yes, PR welcome. Against the master branch.

Reply:
See #22603.